api/v1alpha5/virtualmachine_hardware_types.go

```diff
@@ -229,7 +229,7 @@ type VirtualMachineCdromSpec struct {
 	//   - IDE                -- 4 total (2 per controller)
 	//   - SATA               -- 120 total (30 per controller)
 	//
-	// Defaults to IDE when controllerBusNumber is also omitted; otherwise the
+	// Defaults to IDE when controllerBusNumber is set; otherwise the
```

Comment:
mm I think the original comment is correct if looking at definition of `ControllerBusNumber`?

```
// If this and controllerType are both omitted, the CD-ROM will be attached
// to the  first available IDE controller. If there is no IDE controller
ControllerBusNumber *int32 `json:"controllerBusNumber,omitempty"`
```